src/app/manage-templates/page.tsx

```diff
@@ -43,6 +45,12 @@ async function getAllTemplates(token: string): Promise<ITemplate[]> {
   return templates.data
 }
 
+async function getTokenPayload(token: string): Promise<Token> {
+  const copilotClient = new CopilotAPI(token)
+  const payload = TokenSchema.parse(await copilotClient.getTokenPayload())
+  return payload as Token
```

Comment:
Wouldn't TokenSchema's output be a `Token` by default?
![image](https://github.com/user-attachments/assets/1e939638-eac4-44e4-994f-f2ea193cb96f)